data_structures/stack/infixToPostfix.c

```diff
@@ -0,0 +1,100 @@
+#include<stdio.h>
+#include<string.h>
+
+struct Stack
+{
+    int top;
+    unsigned capacity;
+    int* array;
+};
+struct Stack* createStack(unsigned capacity)
+{
+    struct Stack* stack = (struct Stack*) malloc(sizeof(struct Stack));
+    if(!stack)
+        return NULL;
+    stack->top = -1;
+    stack->capacity = capacity;
+    stack->array = (int*) malloc(stack->capacity * sizeof(int));
+    if(!stack->array)
+        return NULL;
```

Comment:
deallocate memory allocated to ``stack`` before returning to avoid memory leaks.